From 07138e32cae67e0493c6eb8db1b0a7aca90455e7 Mon Sep 17 00:00:00 2001
From: Roy Li <rongqing.li@windriver.com>
Date: Fri, 4 Mar 2016 17:57:38 +0800
Subject: [PATCH] disable SSLv2

a openssl CVE patches openssl-CVE-2016-0800.patch will disable
SSLv2 default build, so we must disable it to pass compilation

Signed-off-by: Roy Li <rongqing.li@windriver.com>
---
 ssl.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ssl.c b/ssl.c
index 2c0db63..5c102a0 100644
--- a/ssl.c
+++ b/ssl.c
@@ -662,11 +662,11 @@ ssl_connection *new_ssl_connection(char *clientpemfile, int sslversion) {
   case SSL_VERSION_AUTO:
     ssl->method = SSLv23_client_method();
     break;
-
+#if 0
   case SSL_VERSION_SSLV2:
     ssl->method = SSLv2_client_method();
     break;
-
+#endif
   case SSL_VERSION_SSLV3:
     ssl->method = SSLv3_client_method();
     break;
-- 
1.9.1

